modules/node_modules/@colony/purser-ledger/staticMethods.js

```diff
@@ -243,7 +244,7 @@ export const signMessage = async ({
        * Also, Flow don't know about Buffer
        */
       /* $FlowFixMe */
-      Buffer.from(message).toString(HEX_HASH_TYPE),
+      Buffer.from(toSign).toString(HEX_HASH_TYPE),
```

Comment:
Ledger seems to like this as well:
![screenshot from 2019-01-20 11-02-56](https://user-images.githubusercontent.com/1193222/51437168-0fb0c200-1ca3-11e9-981e-2c90c5cea150.png)

Produces:
![screenshot from 2019-01-20 11-02-43](https://user-images.githubusercontent.com/1193222/51437170-15a6a300-1ca3-11e9-9e3d-4fa918779b18.png)